.babelrc

```diff
@@ -5,7 +5,7 @@
     "react"
   ],
   "plugins": [
-    "react-relay"
+    "./data/schema"
```

Comment:
It didn't work at first for me when I was trying different things but it does work so I'll put it back